README.md

```diff
@@ -0,0 +1,77 @@
+# cinema-app
+
+**What this application is for and what it can do**
+Primitive demonstration program of service of cinema services and its clients
+
+**Technologies used**
+Program based on the use of:
+- Hibernate 
+- Spring framework
```

Comment:
- Java 11
- MySQL
- Hibernate
- Spring (Core, WEB, Security)
- Apache Tomcat (to run app locally)